Sources/Core/Pointer.swift

```diff
@@ -0,0 +1,56 @@
+// Copyright © 2022 Max Desiatov <max@desiatov.com>.
+// All Rights Reserved.
+// SPDX-License-Identifier: BSD-3
+
+public protocol _Pointer {
+  typealias Distance = Int
+
+  associatedtype Pointee
+
+  var _rawValue: Builtin.RawPointer { get }
+
+  init(_ _rawValue: Builtin.RawPointer)
+}
+
+public extension _Pointer {
+  @_transparent
+  init(_ from: OpaquePointer) {
```

Comment:
I think being more explicit and marking the methods as `public` is better than just marking the extension as `public`

Reply:
I hope a lot of these changes are applied by formatters and linters. Do you have any preference for specific once? I use SwiftLint and SwiftFormat locally, would it make sense to create a separate PR with configurations for those?

Reply:
I would say that swift-format would be better than SwiftLint, but I don't think that swift-format really can format the code in the style that I would prefer.  If you want to add a configuration for swift-format in a separate PR, I am okay with that.